cmd/kubeadm/app/phases/certs/certlist_test.go

```diff
@@ -222,3 +223,110 @@ func parseCertAndKey(basePath string, t *testing.T) (*x509.Certificate, crypto.P
 
 	return parsedCert, certPair.PrivateKey
 }
+
+func Test_createKeyAndCSR(t *testing.T) {
```

Comment:
```suggestion
func TestCreateKeyAndCSR(t *testing.T) {
```
What are the rules for naming test functions?
There is no use of `Test_xxx` in the kubeadm code base.